kodilla-rps/src/main/java/Computer.java

```diff
@@ -0,0 +1,21 @@
+import java.util.Random;
+
+public class Computer {
+
+    private int numberOfWins = 0;
+
+    public ActionType computerAction() {
+        ActionType[] actionTypes = ActionType.values();
+        Random random = new Random();
```

Comment:
on every computer action you create new Random object. More efficient will be to create it only once

Reply:
done